engines/director/detection_tables.h

```diff
@@ -6911,6 +6913,14 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1_l("raveshuttle", "", "Rave Shuttle", "7b0837f93c5de519acdf1feea2a4ccf2", 705673, Common::DE_DEU, 500),
 	WINGAME1_l("raveshuttle", "", "RAVES.EXE", "2e62abdad839e42068afdcd0644d7dcf", 1000911, Common::DE_DEU, 500),
 
+	// German variant of "Robinson Crusoe", published by Tivola
+	// PC/Mac hybrid, but the Projector for the Mac version is embedded into an installer.
+	// The disc contains two versions in two Projector files. The "Standard Version" doesn't use the QTVR features,
+	// while the "Enhanced version" does.
+	WINGAME1_l("robinson",          "Standard Version", "CRUSOE8.EXE", "t:98c98c109102a10f75bd05b76f49a009", 1398759, Common::DE_DEU, 500),
+	WINGAME1_l("robinson",          "Enhanced Version", "CRUSOE16.EXE","t:97c3d5d1472bab61962837b0bf820912", 1398987, Common::DE_DEU, 500),
+	WINGAME1_l("robinson-makingof", "",                 "MAKING.EXE",  "t:4c5428c6b5e493da6fe35daccded70a0", 1410487, Common::DE_DEU, 501),
```

Comment:
This is a bit common, it would be preferable to use an additional file

Reply:
Hmm... but even though the name is the dampening another game - as long as the file hash is unique, then we will have a clean match, right?

Reply:
The problem is that if you add any folder that contains a `MAKING.EXE` file it'll get detected as possible variant of this game (and actually the same can be said for `pqmakingof` which should be updated too if possible)

Reply:
I added additional files to all detection entries now.

Simply adding one for `robinson-makingof` would break the detection of the games themselves, because in this case, the detection entry with 2 valid matches would always "win".